### ISSUE TITLE
Integração GRANITO

### ISSUE THREAD

**Issue by eborguezao** [brunopazz/gateway#2]
Olá, ainda é válida a integração com o gateway GRANITO ? Há 2 anos sem atualização, sabe me informar?

**Comment by brunopazz** [comment]
Sim, esta sim. Faz 2 anos que eles tbm nao fazem nenhuma alteração. Esse plugin já transacionou mais de 10 milhões de Reais

Bruno Paz
